.github/actions/install-python-and-package/action.yml

```diff
@@ -60,7 +60,7 @@ runs:
         # Only way to install msms is through conda
         conda install -c bioconda msms
         # Safest way to install PyTorch and PyTorch Geometric is through conda
-        conda install pytorch torchvision torchaudio cpuonly -c pytorch
+        conda install pytorch torchvision torchaudio cpuonly -c pytorch==2.0.0
```

Comment:
Great! I've also just noticed that in one of my branches. Also I had a question: why is the line 66 needed? (# Install optional pyg dependencies) Shouldn't `conda install pyg -c pyg` take care of the pyg dependencies?

Reply:
Don't know why, but it doesn't